install.rb

```diff
@@ -294,6 +308,16 @@ def prepare_installation
     bindir = RbConfig::CONFIG['bindir']
   end
 
+  if not InstallOptions.localedir.nil?
+    localedir = InstallOptions.localedir
+  else
+    if $operatingsystem == "windows"
+      localedir = File.join(Dir::COMMON_APPDATA, "PuppetLabs", "puppet", "share", "puppet", "locale")
```

Comment:
This one is tricky.  Strictly speaking we should not be putting directories into `Program Files` post MSI installation.  MSI considers these to be "unmanaged" (because it didn't create them) and will not remove them on MSI uninstall. @Iristyle thoughts?